src/main/resources/org/jenkinsci/plugins/workflow/flow/GlobalDefaultFlowDurabilityLevel/help-durabilityHint.html

```diff
@@ -0,0 +1,20 @@
+<div>
+    <p>
+        This setting provides a trade-off between how fast pipelines run and their "durability", or ability to survive unexpected failures while running.
```

Comment:
This and all other doc/help stuff should get reviewed by @rtyler and @bitwiseman at a minimum.

Reply:
@abayer I don't see why, since we've never done this before for simple inline text?   I was planning to get them to review more fleshed out documentation content anyway.